groups/rds/variables.tf

```diff
@@ -42,3 +43,37 @@ variable "team" {
   description = "The name of the team"
   type        = string
 }
+
+variable "db_engine" {
+  default     = "mysql"
+  description = "Database engine"
+  type        = string
+}
+
+variable "db_engine_version" {
+  default = "8.0"
+}
+
+variable "db_instance_class" {
+  default = "db.t4g.small"
+}
```

Comment:
Can we add the `type` and a `description` to these variables for consistency please.